sdk/search/Azure.Search.Documents/src/Indexes/Models/IndexingParametersExtensions.cs

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.Search.Documents.Indexes.Models
+{
+    /// <summary>
+    /// Defines extension methods for the <see cref="IndexingParameters"/> class.
+    /// </summary>
+    public static class IndexingParametersExtensions
```

Comment:
Why are these extension methods?

Reply:
As discussed offline, we'll implement these as a sample for now until we can make these proper properties in the swagger post-GA.

Reply:
We'll make them proper properties post-GA.